src/util/debug.ts

```diff
@@ -1,16 +1,73 @@
-import * as vscode from "vscode";
```

Comment:
This should to be a graph component.  I'm not sure it makes sense to do this refactor unless you're going to take it all the way to a being a graph component.  I'd lean towards just leaving things as they were if you're not comfortable with making a graph component

Reply:
I could easily make it a graph component but decided not to. Nothing in this file needs to be accessed anywhere in the system. Who didn't make sense to me to actually expose it like that.

Reply:
Because it has its own dependencies, so it should them from the graph rather than the hard-coded stuff in extension.  See for example how other graph components access the extension context, other apis, register themselves as disposables, etc

Reply:
Okay I will update it